features/admin.tenants.v1/components/dropdown/tenant-dropdown.tsx

```diff
@@ -17,6 +17,7 @@
  */
 
 import { AsgardeoSPAClient, DecodedIDTokenPayload } from "@asgardeo/auth-react";
+import { CircularProgress } from "@mui/material";
```

Comment:
Let's check if we can use oxygen-ui here?

Reply:
Fixed with 36951c95793531a8fd07e22e7c18fe8e5d176390